pennylane_lightning/lightning_gpu/_measurements.py

```diff
@@ -51,23 +53,35 @@ class LightningGPUMeasurements(LightningBaseMeasurements):
 
     Args:
         qubit_state(LightningGPUStateVector): Lightning state-vector class containing the state vector to be measured.
+        mpi_handler(MPIHandler): MPI handler for PennyLane Lightning GPU device.
+            Provides functionality to run on multiple devices.
+
     """
 
     def __init__(
         self,
         lgpu_state,
+        mpi_handler=None,
     ) -> TensorLike:
 
         super().__init__(lgpu_state)
 
+        if mpi_handler is None:
```

Comment:
Is it safe to create a `MPIHandler` object here?

Reply:
You are right, It is just necessary to pass `use_mpi` bool